### ISSUE TITLE
Docs for `system_info/1` have duplicate entries

### ISSUE THREAD

**Issue by sabiwara** [erlang/otp#7501]
**Describe the bug**
There are duplicate entries in the chunk file docs for `system_info/1`.
This seems due to the fact that the [doc is broken in several sections](https://www.erlang.org/doc/man/erlang.html#system_info-1).

**To Reproduce**

```erlang
{ok, Binary} = file:read_file(ErlPath ++ "/lib/erts-14.0.1/doc/chunks/erlang.chunk"),
Docs = element(7, binary_to_term(Binary)),
9 = length([X || {{function, system_info, 1}, X, _, _, _} <- Docs]).
```

**Expected behavior**

Docs should all be grouped in a single entry.

```erlang
1 = length([X || {{function, system_info, 1}, X, _, _, _} <- Docs]).
```

**Affected versions**
26.0.1

**Additional context**
Originally reported in Elixir: https://github.com/elixir-lang/elixir/issues/12796.

**Comment by garazdawi** [comment]
Hello, thanks for the report!

The `erlang:system_info/1` documentation has been fixed in #7472 which will be part of the next patch (which will also update the website docs).

Regarding multiple entries in the chunk, that works as intended. In Erlang we can document separate clauses of the same MFA using different docs. In the doc chunks this is handles by having multiple entries with the same FA in the list of docs. When a renderer encounters this they can either treat them as separate entries (which is what ExDoc does) or just as one entry (which is what the `h` in the erlang shell does*). For example here is how the text for `erlang:halt/1` looks like:

```
1> h(erlang,halt,1).

  -spec halt(Status :: non_neg_integer()) -> no_return().

  The same as calling halt(Status, []). Example:

    > halt(17).
    os_prompt% echo $?
    17
    os_prompt%

  -spec halt(Abort :: abort) -> no_return().

Since:
  OTP R15B01

  The same as calling halt(abort, []).

  -spec halt(CrashDumpSlogan :: string()) -> no_return().

  The same as calling halt(CrashDumpSlogan, []).
```

Note that there are three entries just as in the [html docs](https://www.erlang.org/doc/man/erlang.html#halt-1).

\* or rather should do, when I checked to verify the behavior I found that a bug had sneaked in while fixing an issue with the `@equiv` metadata. The bug is fixed in #7502.
